src/main/java/io/strimzi/kafka/bridge/http/HttpOpenApiOperationTrace.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.http;
+
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * Represents an OpenApi operation with related logging more restricted to Trace level
+ */
+public abstract class HttpOpenApiOperationTrace extends HttpOpenApiOperation {
```

Comment:
Do we still need this class now? It was only the health endpoints are using it: Couldn't we just default the loggers for those to WARN level in the default log4j config?

Reply:
Actually it's true, I have to figure out how it copes when the bridge is deployed via the CO and if this logging config file is used or we need a change there as well.